syntax = "proto3";

package devices;

message GetDevicesRequest {
}

message Device {
    string id = 1;
    string name = 2;
    string mac_address = 3;
    bool is_active = 4;
    string last_read = 5;
}

message GetDevicesResponse {
    repeated Device devices = 1;
}

service DeviceService {
    rpc GetAllDevices(GetDevicesRequest) returns (GetDevicesResponse);
}
